### DISCUSSION TITLE
Time Field Conditions

### DISCUSSION THREAD

**Discussion by shifuma** [craftcms/cms#14616] [Ideas] [answered]
It seems like Time Fields don't support any conditions. I would like to filter by entries where Time is `empty`.

Is it possible to add that to core, or give me some tips on how to extend conditions to do so? The [docs](https://craftcms.com/docs/4.x/extend/conditions.html) are still quite bare sadly. Do I want a Field condition in this case?

**Comment by brandonkelly** [comment, marked as answer]
Just added simple empty/not-empty condition rule support to Time fields, for Craft 4.9 and 5.0 (9552142f2a84070249b6d1772d3df5f41b5df357).

If you need it right away, you can update to 4.9 early by changing your `craftcms/cms` requirement in composer.json to `4.9.x-dev as 4.9.0-alpha`, and running `composer update`.

**Reply by shifuma** [reply]
Amazing, thank you!

**Reply by brandonkelly** [reply]
Craft 4.9 (and 5.0) are out now with that fix.
